source/connecting/oidc-to-oidc.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-conn-oidc2oidc:
+
+================================================
+Connect Two Clusters that Support OpenID Connect
+================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+The source and destination clusters must use MongoDB 7.0 or later.
```

Comment:
OIDC is only supported in the go driver in 7.0 or later @pmeredit?

If so we should say 7.0 and not later.

Reply:
Later is fine. It will definitely be supported in 8.0 also.

Reply:
@rohshar @pmeredit ok, so keep as "MongoDB 7.0 or later"?

Reply:
Can mongosync migrate from a self-managed, non-OIDC 6.0 to an OIDC-authenticated 7.0?